integreat_cms/api/v3/pages.py

```diff
@@ -20,25 +20,34 @@ def transform_page(page_translation):
     :param page_translation: single page translation object
     :type page_translation: ~integreat_cms.cms.models.pages.page_translation.PageTranslation
 
+    :raises ~django.http.Http404: HTTP status 404 if a parent is archived
+
     :return: data necessary for API
     :rtype: dict
     """
+    fallback_parent = {
+        "id": 0,
+        "url": None,
+        "path": None,
+    }
+
     parent_page = page_translation.page.parent
```

Comment:
```suggestion
    parent_page = page_translation.page.cached_parent
```

Do you think we could add this cached value with this PR again? With cacheops disabled, I think this will work fine and should reduce the number of queries tremendously.

Reply:
Sure.